Cargo.toml

```diff
@@ -8,8 +8,12 @@ default = []
 dev = ["clippy"]
 
 [[bin]]
-name = "example"
-path = "example/simple_example.rs"
+name = "example_hyper"
```

Comment:
please follow http://doc.crates.io/guide.html#project-layout, maybe we don't need bin for examples.